include/zephyr/bluetooth/conn.h

```diff
@@ -1180,6 +1273,35 @@ enum bt_conn_auth_keypress {
 	BT_CONN_AUTH_KEYPRESS_ENTRY_COMPLETED = 0x04,
 };
 
+/** @brief Get the connection interval in microseconds
+ *
+ * @param conn_info pointer to a @ref bt_conn_info object
+ * @return The connection interval in microseconds
+ */
+static inline uint32_t bt_conn_le_get_interval_us(const struct bt_conn_info *conn_info)
```

Comment:
I think "action" should be at the end of the function call. There are more of this.

```suggestion
static inline uint32_t bt_conn_le_interval_us_get(const struct bt_conn_info *conn_info)
```